### PR TITLE
Cleanup deployed prs

### PR DESCRIPTION
**Pull request by kim0**
Resolves https://github.com/brave/devops/issues/8849

### CODE REVIEW

**Review by mihaiplesa — APPROVED** [review]

File: .github/workflows/cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: cleanup
+
+on:
+  schedule:
+    - cron: '30 5 * * 1'
+  workflow_dispatch:
+  push:
+    branches:
+      - 'aka/cleanup_prs'
```
Comment (AlanBreck) [inline .github/workflows/cleanup.yml:9]: Does this need to be removed prior to merge?

**Review by AlanBreck — APPROVED** [review]

**Review by fallaciousreasoning — APPROVED** [review]
LGTM - out of interest we can't set this up to run when branches merge?

**Comment by kim0** [comment]
@fallaciousreasoning That can certainly work too, I opted for this approach bec we have to ping GH about every open PR which can get kinda expensive on busy repos. This is an approach I used before elsewhere and worked well so I went with it.

File: .github/workflows/cleanup.yml (hunk 1 above)
Comment (kim0) [reply .github/workflows/cleanup.yml]: Correct. Removed

**Review by mihaiplesa — APPROVED** [review]

**Pull request merged**